docs/config.rst

```diff
@@ -1,11 +1,606 @@
 Configuring CumulusCI
 =====================
 
-cumulusci.yml
--------------
+CumulusCI merges multiple `YAML <https://yaml.org/>`_ files that allow for configuration to occur at several distinct scopes. All of these files have the same name, ``cumulusci.yml``, but live in different locations on the file system. (See `configuration scopes`_ for further detail.)
 
-Global overrides
-----------------
+The `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file contains the default configuration stored within CumulusCI itself. This is the location of the standard library of tasks and flows that come with CumulusCI.
 
-Local overrides
----------------
+The project ``cumulusci.yml`` file is located in the ``project`` diretory. You can override aspects of the universal ``cumulusci.yml`` file here by defining project dependencies, configuring new tasks and flows, customizing standard tasks and flows to meet your project's needs, and so much more! 
+
+    .. Important:: When ``cumulusci.yml`` is referenced in this documentation, it refers to the project ``cumulusci.yml`` file unless otherwise specified.
+
+
+
+``cumulusci.yml`` Structure
+---------------------------
+
+There are six top-level sections of a ``cumulusci.yml`` file.
+
+* ``project``: This section is largely generated by running ``cci project init``. It contains information pertaining to the project's package information (if any) and GitHub information.
+
+* ``tasks``: This section defines the tasks that are available to run in your project. See `task configurations`_ for more info on the types of configurations that can be made in this section.
+
+* ``flows``: This section defines the flows that are available to run in your project. See `flow configurations`_ for more info on the types of configurations that can be made in this section.
+
+* ``sources``: This section references tasks and flows from another CumulusCI project on GitHub. See the `use tasks and flows from a different project`_ section for more information.
+
+* ``orgs``: This section defines the scratch org configurations that are available for your project. See `scratch org configurations`_ for more information on the types of configurations that can be made in this section.
+
+* ``plans``: This section contains any custom plans purposely defined to install your project into a customer org. For more information, see `configuring plans in MetaDeploy<TODO>`_.
+
+A complete list of values that can be used in each section can be found in `cumulusci.yml reference<TODO>`_.
+
+
+
+Task Configurations
+-------------------
+
+Task configurations are defined under the ``tasks`` section of your ``cumulusci.yml`` file. Each task configuration defines a task which can be run using the ``cci task run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
```

Comment:
```suggestion
Each task configurations in the ``tasks`` section of your ``cumulusci.yml`` file defines a task that can be run using the ``cci task run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
```
The classic "that versus which" usage situation. :) I think we mean "that" here but check https://confluence.internal.salesforce.com/display/DOCTEAM/that%2C+which and https://www.quickanddirtytips.com/education/grammar/which-versus-that-0 to be sure.